teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(name, age)
```

Comment:
- [ ] Please add `parent_permission` to the constructor parameters of the `Teacher` class and pass it down to the `Person` class using `super()`.